### ISSUE TITLE
Filtering for article subjects, titles & texts via reporting profile does not work

### ISSUE THREAD

**Issue by MrGeneration** [zammad/zammad#3034]
### Infos:

* Used Zammad version: 3.3
* Installation method (source, package, ..): any
* Operating system: any
* Database + version: any
* Elasticsearch version: any
* Browser + version: Google Chrome latest
* Ticket-ID: #1073785, #1078940

### Affected attributes:

* article-subject
* article-text
* ticket-title

### Expected behavior:

Zammad will return a graph of tickets where the subject matches (if configured) and thus will display what the reporting profile configuration already suggests.

### Actual behavior:

Zammad does return empty responses when filtering for e.g. `Subject contains delivery`. While the reporting fails, the report value configuration correctly suggests tickets for that search filter.

The same applies for the article text.

No errors can be found within production log or developer console.

```
I, [2020-04-23T09:16:27.813609 #594-47022727786820]  INFO -- : Completed 200 OK in 49ms (Views: 0.2ms | ActiveRecord: 15.2ms)
I, [2020-04-23T09:16:27.833196 #594-47022734227600]  INFO -- : Started POST "/api/v1/reports/sets" for 192.168.2.122 at 2020-04-23 09:16:27 +0200
I, [2020-04-23T09:16:27.836052 #594-47022734227600]  INFO -- : Processing by ReportsController#sets as JSON
I, [2020-04-23T09:16:27.836108 #594-47022734227600]  INFO -- :   Parameters: {"metric"=>"count", "year"=>2020, "month"=>4, "week"=>17, "day"=>23, "timeRange"=>"year", "profiles"=>{"2"=>true}, "backends"=>{"count::created"=>true, "count::closed"=>true, "count::backlog"=>true, "create_channels::phone_in"=>true, "create_channels::phone_out"=>true, "create_channels::email_in"=>true, "create_channels::email_out"=>true, "create_channels::web_in"=>true, "create_channels::twitter_in"=>true, "create_channels::twitter_out"=>true, "communication::phone_in"=>true, "communication::phone_out"=>true, "communication::email_in"=>true, "communication::email_out"=>true, "communication::web_in"=>true, "communication::twitter_in"=>true, "communication::twitter_out"=>true, "count::movedout"=>true, "count::movedin"=>true, "count::reopened"=>true, "count::first_solution"=>true}, "downloadBackendSelected"=>"count::created"}
I, [2020-04-23T09:16:27.934196 #594-47022734227600]  INFO -- : # curl -X get "http://localhost:9200/zammad_production_ticket/_doc/_search"
I, [2020-04-23T09:16:27.943380 #594-47022734227600]  INFO -- : # 200
I, [2020-04-23T09:16:27.944814 #594-47022734227600]  INFO -- : Completed 200 OK in 109ms (Views: 0.2ms | ActiveRecord: 1.1ms)
```

![image](https://user-images.githubusercontent.com/6549061/80070497-53981400-8543-11ea-8579-5323276a9d59.png)

During configuration of the reporting profile Zammad correctly suggests ticket it should later show as well.
![image](https://user-images.githubusercontent.com/6549061/80070775-d15c1f80-8543-11ea-8e7c-6c2a0f984263.png)

### Steps to reproduce the behavior:

* ensure to have a ticket with a subject that contains a string you know, e.g. "delivery" - if you can't, send a mail to Zammad 
* After receiving the ticket, create a new reporting profile
  * configure `Subject contains delivery` (replace delivery by whatever you have)
  * Zammad will show suggestions that fit your condition
  * Submit this reporting profile
* Go to reporting and select your newly created reporting profile filter
* find an empty graph

You can repeat the above steps with "article text" which doesn't work as well.

Yes I'm sure this is a bug and no feature request or a general question.

**Comment by egorky** [comment]
I can confirm this too happens with tags.
When creating the profile, the right ticket with one of the tags show in the preview:
![image](https://user-images.githubusercontent.com/54412925/89701754-8651f680-d8ff-11ea-9eab-2fdc12d01afb.png)
But on the reporting page one that doesn't have any tag shows up:
![image](https://user-images.githubusercontent.com/54412925/89701793-d7fa8100-d8ff-11ea-9ceb-c2c7ea22f4c5.png)

**Comment by rolfschmidt** [comment]
https://github.com/zammad/zammad/issues/3243

**Comment by rolfschmidt** [comment]
Hi,
 
since the rework of #3243 this should be fixed also with zammad 4.0. If you have still any issues let me know.
